<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PB_ActionSelectItemDrawableGVOptions</class>
 <widget class="QWidget" name="PB_ActionSelectItemDrawableGVOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>36</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sélection</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButtonSelectOne">
     <property name="toolTip">
      <string>Sélectionner un élément</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor.png</normaloff>:/icons/cursor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonSelectMulti">
     <property name="toolTip">
      <string>Sélectionner de multiple éléments</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor_multi.png</normaloff>:/icons/cursor_multi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAddOne">
     <property name="toolTip">
      <string>Ajouter un élément à la sélection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor_add.png</normaloff>:/icons/cursor_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAddMulti">
     <property name="toolTip">
      <string>Ajouter de multiple éléments à la sélection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor_multi_add.png</normaloff>:/icons/cursor_multi_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonRemoveOne">
     <property name="toolTip">
      <string>Supprimer un élément de la sélection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor_remove.png</normaloff>:/icons/cursor_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonRemoveMulti">
     <property name="toolTip">
      <string>Supprimer de multiple éléments de la sélection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/icons/cursor_remove_multi.png</normaloff>:/icons/cursor_remove_multi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupSelection</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonItems">
     <property name="toolTip">
      <string>Action sur les ItemDrawable</string>
     </property>
     <property name="text">
      <string>Items</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupType</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonPoints">
     <property name="toolTip">
      <string>Action sur les points</string>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupType</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonFaces">
     <property name="toolTip">
      <string>Action sur les faces</string>
     </property>
     <property name="text">
      <string>Faces</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupType</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonEdges">
     <property name="toolTip">
      <string>Action sur les edges</string>
     </property>
     <property name="text">
      <string>Edges</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupType</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupType"/>
  <buttongroup name="buttonGroupSelection"/>
 </buttongroups>
</ui>
